src/api/app/models/bs_request.rb

```diff
@@ -51,11 +51,13 @@ class SaveError < APIException
   has_many :comments, as: :commentable, dependent: :delete_all
   has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
   has_many :review_history_elements, through: :reviews, source: :history_elements
+  belongs_to :user
```

Comment:
I think  belongs_to :creator, class_name: "User" would be nicer. Sorry for adding more work on your shoulders:-)